app/views/mountain_view/styleguide/show.html.erb

```diff
@@ -12,15 +12,24 @@
     <% if @component.component_stubs? %>
       <% @component.component_stubs.each_with_index do |component_properties, index| %>
           <div class="mv-component">
-            <div class="mv-component__item">
+            <% if @component.example_classes(component_properties).blank? %>
```

Comment:
We can get rid of this if and always interpolate the extra classes. If they are blank, it just won't add anything but a blank space which is still valid HTML.